helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -310,6 +310,12 @@ image:
   # The name of the pull secret to attach to the respective serviceaccount used to pull the image
   pullsecretName:
 
+# Environment variable for daemonset
+# Remove SSL_CERT_FILE environment variable when using OS CA file
+environmentVar:
```

Comment:
how to remove a default value set in the chart?

Reply:
you can leave environmentVar:  to be blank.

Reply:
at least not by passing empty environmenVar with values.yml:

```
$> cat values.yml 
splunk-kubernetes-logging:
  # Environment variable for daemonset
  # Remove SSL_CERT_FILE environment variable when using OS CA file
  environmentVar:
$> helm template --values ./values.yml sck  splunk-connect-for-kubernetes/  | grep -C4 SSL
              secretKeyRef:
                name: splunk-kubernetes-logging
                key: splunk_hec_token
          
          - name: SSL_CERT_FILE
            value: /fluentd/etc/splunk/hec_ca_file
        resources:
          
            requests:
--
...
```

Reply:
setting `null` also doesn't work:

```
$> cat values.yml 
splunk-kubernetes-logging:
  # Environment variable for daemonset
  # Remove SSL_CERT_FILE environment variable when using OS CA file
  environmentVar: null
$> helm template --values ./values.yml sck  splunk-connect-for-kubernetes/  | grep -C4 SSL
              secretKeyRef:
                name: splunk-kubernetes-logging
                key: splunk_hec_token
          
          - name: SSL_CERT_FILE
            value: /fluentd/etc/splunk/hec_ca_file
        resources:
          
            requests:
--
```

Reply:
also passing empty dict fails:

```
$> cat values.yml 
splunk-kubernetes-logging:
  # Environment variable for daemonset
  # Remove SSL_CERT_FILE environment variable when using OS CA file
  environmentVar: {}
$> helm template --values ./values.yml sck  splunk-connect-for-kubernetes/  | grep -C4 SSL
coalesce.go:203: warning: destination for environmentVar is a table. Ignoring non-table value [map[name:SSL_CERT_FILE value:/fluentd/etc/splunk/hec_ca_file]]
coalesce.go:163: warning: skipped value for environmentVar: Not a table.
coalesce.go:203: warning: destination for environmentVar is a table. Ignoring non-table value [map[name:SSL_CERT_FILE value:/fluentd/etc/splunk/hec_ca_file]]
coalesce.go:163: warning: skipped value for environmentVar: Not a table.
Error: YAML parse error on splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/daemonset.yaml: error converting YAML to JSON: yaml: line 58: did not find expected '-' indicator

Use --debug flag to render out invalid YAML
```

Reply:
it should be empty array `[]` instead of `{}` 
Are you getting that env file from logging daemonset pod?